docs/reference/transform/apis/get-transform.asciidoc

```diff
@@ -65,6 +65,12 @@ include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=from-transforms]
 (Optional, integer)
 include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=size-transforms]
 
+`exclude_generated`::
+(Optional, boolean)
+Indicates if certain fields should be removed from the configuration on
```

Comment:
"certain fields" is a bit vague 

"Exclude fields automatically added to the configuration when the transform is created."